submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/factory/AllInOneSubmissionJson.kt

```diff
@@ -21,7 +23,7 @@ fun allInOneSubmissionJson(accNo: String) = jsonObj {
         },
         {
             "name" to "ReleaseDate"
-            "value" to "2021-02-12"
+            "value" to OffsetDateTime.now().toStringDate()
```

Comment:
IMO using evaluated expression in itest make it harder to follow and also may generate issues depending of computer time zone and hour. I think using fixed value make easier to follow as is super clear for example that 2021 is the past of 2040 is in the future.

Reply:
I understand your point but after adding the restriction of not allowing release dates in the past, the only way to make a submission public right away would be to use the same day. I also thought about mocking the date but then it wouldn't be a real iTest.